src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -320,7 +320,9 @@ export class ExtensionsListView extends CollapsibleView {
 				}
 
 				return this.extensionsWorkbenchService.queryGallery(assign(options, { names, pageSize: names.length }))
-					.then(pager => new PagedModel(pager));
+					.then(pager => this.addRecommendationInfo(new PagedModel(pager || []), {
+						source: 'fileType'
+					}));
```

Comment:
@sandy081 I have added a `updateTelemetryData()` method on the `Extensions`. This will help in the future when we want to add extra info to track for extensions.